project/VersionSyncCheckPlugin.scala

```diff
@@ -19,18 +22,20 @@ object VersionSyncCheckPlugin extends AutoPlugin {
     versionSyncCheck := versionSyncCheckImpl.value
   )
 
-  final case class Entry(name: String, expectedVersion: String, valName: String)
+  final case class Entry(name: String, expectedVersion: String, valName: String, skipCheck: Boolean = false)
```

Comment:
I choose to enrich the model because we probably want to have other dynamic versions in the future (eg: Play).